### DISCUSSION TITLE
What js can I safely disable on pages that don't have donation features

### DISCUSSION THREAD

**Discussion by mb0x88** [impress-org/givewp#7317] [Developer Q&A]
<img width="288" alt="Screenshot 2024-03-20 at 11 32 49" src="https://github.com/impress-org/givewp/assets/106029773/d1e850dc-68ac-43a3-8a98-d2518947f666">

I would like to not load donation js unless it is necessary for certain pages to optimise our site, do you have a guide for this?  I'm assuming this js can be disabled on pages that don't access donation info or do donation processes.  Can they also disabled on the backend pages?

**Comment by jonwaldstein** [comment]
Hey @mb0x88 

Thanks for reaching out!

We do not recommend disabling GiveWP scripts.  If you find scripts that are being loaded where you think they should not be - please let us know so we can try and resolve on our end.
